scripts/nymms_api

```diff
@@ -10,4 +12,8 @@ logger = cli.setup_logging(args.verbose)
 config.load_config(args.config)
 
 from nymms.api import routes
+
+for plugin in config.settings.get('api', []).get('plugins', []):
```

Comment:
Yeah, I guess the bigger issue is that it'd happen silently, since the SimpleDBBackend creates the domain automatically.  I'll think about it some more, but for now this is better than not having it :)